languages/rust/exercises/concept/enums/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+// TODO define enum variants
+pub enum LogLevel {}
```

Comment:
Given that we explicitly refer to each variant in the test cases, it's only fair to define them here. The student won't get much from wading through compiler errors to figure out what they should write here that they wouldn't get from reading a properly predefined enum.

Reply:
If they don't write the enum, then (if I'm reading the example.rs correctly), the only code they write to solve the exercise is an if/else. Based on the idea that learning-by-doing is the most effective way to learn, I think that we want someone to be learning how to write the actual enum (and struggling through that if they do), rather than them reading it and hoping they remember it.